common/cmake/devices.cmake

```diff
@@ -929,6 +929,191 @@ function(ADD_FPGA_TARGET_BOARDS)
   endforeach()
 endfunction()
 
+function(ADD_BITSTREAM_TARGET)
```

Comment:
This needs a block comment explaining what it does, how to use it, and what the arguments are for.  Thanks!